pkg/services/services.go

```diff
@@ -30,6 +31,32 @@ type OAuthClient interface {
 	Configure() error
 }
 
+// AuthenticationTokenContainerEncoded defines the interface for Authentication Token Containers services
+type AuthenticationTokenContainerEncoded interface {
+	// Decode accepts a raw token container encoded as a string and decodes it into a NutsAuthenticationTokenContainer
+	Decode(rawTokenContainer string) (*NutsAuthenticationTokenContainer, error)
+
+	// Encode accepts a NutsAuthenticationTokenContainer and encodes in into a string
+	Encode(authTokenContainer *NutsAuthenticationTokenContainer) (string, error)
```

Comment:
why pass it by reference? Now the caller doesn't know whether this function will modify the struct